airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -120,6 +118,7 @@
   private static final int STDIN_REMOTE_PORT = 9001;
   private static final Map<String, String> AIRBYTE_POD_LABELS = Map.of("airbyte", "worker-pod");
 
+  private final String processRunnerHost;
```

Comment:
looks like we don't need to store this as class state and having it within the function is sufficient.